task/ab-load/0.1/ab-load.yaml

```diff
@@ -0,0 +1,161 @@
+            local concurrency=${3:-10}
+            local timeout=${4:-10}
+            local additionnal=${5}
+            echo "= Start http test POST ${AB_URL}"
+            echo "  - ${size} requests"
+            echo "  - ${concurrency} concurrent jobs"
+            ab -n "$size" \
+               -c "$concurrency" \
+               -s "$timeout" \
+               -p "$file" \
+               -T '${type}; charset=UTF-8' \
+            $additionnal "$url"
+          }
+
+          # execute the AB benchmark for PUT load
+          function abPUT {
+            local file=${AB_FILE}
+            local type=${AB_TYPE:-application/x-www-form-urlencoded}
+            local url=${1:-myurl}
+            local size=${2:-1000}
+            local concurrency=${3:-10}
+            local timeout=${4:-10}
+            local additionnal=${5}
+            echo "= Start http test PUT ${AB_URL}"
+            echo "  - ${size} requests"
+            echo "  - ${concurrency} concurrent jobs"
+            ab -n "$size" \
+               -c "$concurrency" \
+               -s "$timeout" \
+               -u "$file" \
+               -T '${type}; charset=UTF-8' \
+            $additionnal "$url"
+          }
+
+          # execute the AB benchmark for DELETE load
+          function abDELETE {
+            abDefault DELETE "$url" "$size" "$concurrency" "$timeout" "$additionnal"
+          }
+
+          if [[ $AB_METHOD == 'DELETE' ]]
+          then
+              abDELETE "$AB_URL" "$AB_SIZE" "$AB_CONCURRENCY" "$AB_TIMEOUT" "$AB_ADDITIONNAL_FLAGS"
+          elif [[ $AB_METHOD == 'POST' ]]
+          then
+              abPOST "$AB_URL" "$AB_SIZE" "$AB_CONCURRENCY" "$AB_TIMEOUT" "$AB_ADDITIONNAL_FLAGS"
+          elif [[ $AB_METHOD == 'PUT' ]]
+          then
+              abPUT "$AB_URL" "$AB_SIZE" "$AB_CONCURRENCY" "$AB_TIMEOUT" "$AB_ADDITIONNAL_FLAGS"
+          else
+              abGET "$AB_URL" "$AB_SIZE" "$AB_CONCURRENCY" "$AB_TIMEOUT" "$AB_ADDITIONNAL_FLAGS"
+          fi
+      env:
+        - name: AB_URL
+          value: $(params.url)
+        - name: AB_SIZE
+          value: $(params.size)
+        - name: AB_CONCURRENCY
+          value: $(params.concurrency)
+        - name: AB_TIMEOUT
+          value: $(params.timeout)
+        - name: AB_METHOD
+          value: $(params.method)
+        - name: AB_ADDITIONNAL_FLAGS
+          value: $(params.additionnalFlags)
+        - name: AB_FILE
+          value: /tmp/postfile/content
+        - name: AB_TYPE
+          value: $(params.type)
+      resources: {}
+  workspaces:
+    - name: body-content
+      description: >-
+        Data to send to the endpoint must have a `content` key with content ready to send to the endpoint.
+        Could be from a `configMap`, a `secret` or a `persistentVolumeClaim` as soon as they have a `content` key.
+      mountPath: /tmp/postfile
+      optional: true
+  results:
+    - name: report
+      description: The final report of the http load test
```

Comment:
we aren't writing anything to the results in the above script. Can we remove it or modify the above script to produce the results?